src/test/miner_tests.cpp

```diff
@@ -209,11 +209,11 @@ BOOST_AUTO_TEST_CASE(CreateNewBlock_validity)
 
```

Comment:
The initialization code in the actual miner has changed in the following way:

```diff

-                    // Create solver and initialize it.
-                    equi eq(1);
+                    equi& eq = *peq;
+                    // initialize solver
```

but in this (commented-out) code above, it has not been changed by this PR. Are there any other places that need to be changed?